### ISSUE TITLE
Installation issue

### ISSUE THREAD

**Issue by Itthakur** [ECE49595-Team-6/EnhancifyInstall#11]
I had the same problem as the issue already mentioned but with a slight change. I did the same while running it, copying it from notepad and running in Powershell but in the last, I got a message saying "Enhancify doesn't have index.js"
![Screenshot 2024-11-18 222712](https://github.com/user-attachments/assets/eb061da7-a565-4dea-88ff-1a52313aff9c)

**Comment by Itthakur** [comment]
Also I extracted into my dowloads file, do I have to extract in the folder where spicetify is?

**Comment by rrk0804** [comment]
Yes. Basically, the installation script moves the Enhancify folder from the downloads folder to the Spicetify folder. You can try manually moving it to the custom apps folder of your Spicetfy folder. Then extract the zip file there

**Comment by Itthakur** [comment]
Still no progress after extracting it to the spicetify folder
